rules/windows/builtin/ntlm/win_susp_ntlm_brute_force.yml

```diff
@@ -0,0 +1,31 @@
+title: NTLM Brute Force
+status: experimental
```

Comment:
`id: 9c8acf1a-cbf9-4db6-b63c-74baabe03e59`
I took one from https://www.uuidgenerator.net/version4

Reply:
Thanks @frack113 I have made the change.